pom.xml

```diff
@@ -39,27 +38,15 @@
       <artifactId>github-api</artifactId>
       <version>1.67</version>
     </dependency>
-    <dependency>
+    <dependency><!-- minimun required version to work with Workflow 1.4 GitSCM step-->
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <version>2.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jmock</groupId>
-      <artifactId>jmock-junit4</artifactId>
-      <version>2.5.1</version>
-      <scope>test</scope>
+      <version>2.3.5</version>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jgit</groupId>
-      <artifactId>org.eclipse.jgit</artifactId>
-      <version>0.12.1</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>multiple-scms</artifactId>
-      <version>0.2</version>
-      <optional>true</optional>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.6.6</version>
```

Comment:
```org.slf4j:slf4j-api:1.6.6``` was in the classpath because ```org.jenkins-ci.plugins:git:2.0``` got it transitively, it was removed on ```org.jenkins-ci.plugins:git:2.3.5```. Since ```slf4j``` is directly used in the plugin code we have to add it explicitly (or stop using it, but this is out of the PR scope).